src/injected/selector-map-helper.ts

```diff
@@ -69,6 +69,7 @@ export class SelectorMapHelper {
                 VisualizationType.Landmarks,
                 VisualizationType.TabStops,
                 VisualizationType.Color,
+                VisualizationType.NeedsReview,
```

Comment:
We don't need to do this right now but I wonder if we can just use an if else instead at the top and get rid of this function. Basically have the logic:

```
if this test is an assement
    // do something for assessments
else
    // call getFilteredSelectorMap
```

Reply:
Made a note for future work to adjust this